lib/fastlane/plugin/wpmreleasetoolkit/actions/configure/configure_download_action.rb

```diff
@@ -0,0 +1,94 @@
+require 'fastlane/action'
+require 'fastlane_core/ui/ui'
+require 'fileutils'
+
+module Fastlane
+  module Actions
+    class ConfigureDownloadAction < Action
+      def self.run(params = {})
+
+        UI.message "Running Configure Download"
+
+        # If the `~/.mobile-secrets` repository doesn't exist
+        unless File.directory?("#{Dir.home}/.mobile-secrets")
+
+          choice = UI.select("No local secret store exists. Please choose an option: ", [
+              "Install the sample data (default)",
+              "Install from a git repository"
+          ])
+
+          # If they enter anything other than "2", just do the default action
+          case choice
+              when "2" then install_from_git_repository 
+              else install_sample_data
+          end
+        else
+          update_repository # If the repo already exists, just update it
+        end
+      end
+
+      # Install the sample repository
+      def self.install_sample_data
+
+        UI.message "Installing from sample data"
+
+        clone_repository("https://github.com/jkmassel/mobile-secrets-repository")
```

Comment:
We should make this something that falls under the WP repos, rather than my personal. It's just there for testing right now :)